cmd/adwatchd/integration_tests/adwatchd_service_test.go

```diff
@@ -277,3 +277,12 @@ func TestServiceConfigFlagUsage(t *testing.T) {
 		})
 	}
 }
+
+func TestServiceWithNoNameFails(t *testing.T) {
+	app := commands.New(commands.WithServiceName(""))
+
+	changeAppArgs(t, app, "", "service", "status")
+	err := app.Run()
+
+	require.Error(t, err, "Service with empty name should fail to create")
+}
```

Comment:
I would organize the blocks as:
```suggestion
func TestServiceWithNoNameFails(t *testing.T) {
	app := commands.New(commands.WithServiceName(""))
	changeAppArgs(t, app, "", "service", "status")

	err := app.Run()
	require.Error(t, err, "Service with empty name should fail to create")
}
```
Could be just a biased opinion, but I feel like it makes more sense considering what's happening